app/Http/Controllers/Auth/ForgotPasswordController.php

```diff
@@ -13,6 +13,28 @@ class ForgotPasswordController extends Controller
 {
     use SendsPasswordResetEmails;
 
+    /**
+     * Send a reset link to the given user.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @return \Illuminate\Http\RedirectResponse|\Illuminate\Http\JsonResponse
+     */
+    public function sendResetLinkEmail(Request $request)
```

Comment:
Why is this here, it appears to be exactly same as the function defined in the trait?